index.js

```diff
@@ -0,0 +1,50 @@
+
+function validateLineup(lineup) {
+  // returns false when the lineup includes too many players from a single team
+
+  const getPlayers = lineup
+    .map(lineup => lineup.teamId)
+    .sort((a, b) => a - b)
+  let count = 0
+
+  for (let i = 0; i < getPlayers.length; i++) {
+    if (count >= 2) return false
+    if (getPlayers[i] === getPlayers[i + 1]) count++
+  }
```

Comment:
This approach is looping through the lineup three times. You could instead use reduce to loop once, reformatting the data like we did in class and then check a much smaller array of values for ones that violate the counts. See the solution branch.